pkg/utils/topology/tso.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 PingCAP, Inc. Licensed under Apache-2.0.
+
+package topology
+
+import (
+	"context"
+	"encoding/json"
+	"sort"
+
+	"github.com/pingcap/tidb-dashboard/pkg/pd"
+	"github.com/pingcap/tidb-dashboard/util/distro"
+	"github.com/pingcap/tidb-dashboard/util/netutil"
+)
+
+func FetchTSOTopology(ctx context.Context, pdClient *pd.Client) ([]TSOInfo, error) {
+	nodes := make([]TSOInfo, 0)
+	data, err := pdClient.WithoutPrefix().SendGetRequest("pd/api/v2/ms/members/tso")
```

Comment:
miss the `/` at the begining, else it reports the following error:

![image](https://github.com/pingcap/tidb-dashboard/assets/1284531/e816f6e0-be69-4718-9650-a6eb4da1cdf0)